ietf/static/js/agenda_timezone.js

```diff
@@ -8,272 +7,300 @@
-            end.fromNow();
+        if (end.isBefore()) {
+            notice = "Event ended " + end.fromNow();
+        } else if (start.isAfter()) {
+            notice = "Event will start " + start.fromNow();
+        } else {
+            notice = "Event started " + start.fromNow() + " and will end " +
+                end.fromNow();
+        }
+        return '<span class="tooltipnotice">' + notice + '</span>';
     }
-    return '<span class="tooltipnotice">' + notice + '</span>';
-}
 
-// Format tooltip table
-function format_tooltip_table(start, end) {
-    var current_timezone = get_current_tz_cb();
-    var out = '<div class="text-start"><table class="table table-sm"><tr><th></th><th>Session start</th><th>Session end</th></tr>';
-    if (window.meeting_timezone !== "") {
-        out += '<tr><th class="timehead">Meeting timezone</th><td class="text-nowrap">' +
-            format_time(start, window.meeting_timezone, 0) + '</td><td class="text-nowrap">' +
-            format_time(end, window.meeting_timezone, 0) + '</td></tr>';
-    }
-    out += '<tr><th class="timehead">Local timezone</th><td class="text-nowrap">' +
-        format_time(start, local_timezone, 0) + '</td><td class="text-nowrap">' +
-        format_time(end, local_timezone, 0) + '</td></tr>';
-    if (current_timezone !== 'UTC') {
-        out += '<tr><th class="timehead">Selected Timezone</th><td class="text-nowrap">' +
-            format_time(start, current_timezone, 0) + '</td><td class="text-nowrap">' +
-            format_time(end, current_timezone, 0) + '</td></tr>';
+    // Format tooltip table
+    function format_tooltip_table(start, end) {
+        const current_timezone = get_current_tz_cb();
+        let out = '<div class="text-start"><table class="table table-sm"><tr><th></th><th>Session start</th><th>Session end</th></tr>';
+        if (window.meeting_timezone !== "") {
+            out += '<tr><th class="timehead">Meeting timezone</th><td class="text-nowrap">' +
+                format_time(start, window.meeting_timezone, 0) + '</td><td class="text-nowrap">' +
+                format_time(end, window.meeting_timezone, 0) + '</td></tr>';
+        }
+        out += '<tr><th class="timehead">Local timezone</th><td class="text-nowrap">' +
+            format_time(start, local_timezone, 0) + '</td><td class="text-nowrap">' +
+            format_time(end, local_timezone, 0) + '</td></tr>';
+        if (current_timezone !== 'UTC') {
+            out += '<tr><th class="timehead">Selected Timezone</th><td class="text-nowrap">' +
+                format_time(start, current_timezone, 0) + '</td><td class="text-nowrap">' +
+                format_time(end, current_timezone, 0) + '</td></tr>';
+        }
+        out += '<tr><th class="timehead">UTC</th><td class="text-nowrap">' +
+            format_time(start, 'UTC', 0) + '</td><td class="text-nowrap">' +
+            format_time(end, 'UTC', 0) + '</td></tr>';
```

Comment:
This will create a merge conflict with https://github.com/ietf-tools/datatracker/pull/3662, which also changes this. Can deal with that with a rebase though.

Reply:
Ok. Hopefully it won't be too painful as most of this is just an additional level of indentation.

Reply:
I merged ietf-tools#3662 upstream and then into this branch. I believe it went smoothly.